assets/scripts/users/authentication.js

```diff
@@ -192,26 +192,13 @@ async function fetchFreshTokens (refreshToken) {
       throw response
     }
 
-    const json = await response.json()
-    const { token } = json
-
-    receiveFreshTokens({ token })
     return
   } catch (error) {
-    errorReceiveFreshTokens(error)
+    errorRefreshLoginToken(error)
   }
 }
 
-function receiveFreshTokens (freshTokens) {
```

Comment:
If this is removed, then what handles a successful response from `refreshLoginToken()`? Do we no longer need to dispatch or save any other sign in data in addition to `freshTokens`?

Reply:
Ah I see - the fetch request itself returns cookies. that's all we need.